### PR TITLE
Add Spanish updates

### PR DESCRIPTION
**Pull request by andycyca**
- 46: Changed to potentially better (more collocquial) usage of the phrase
- Changed ampersand to the Spanish word "y"
- Changed references from "colorblind people" ("daltónico/a") to "people with colorblindness" ("personas con daltonismo")
- 110: Changed to Spanish word Espacio
- Changed changelog from "registro" to "historial" as to better convey the time-sensitive nature of logs
- Changed the Spanish word "resetear" so that "resetting" is consistently referred to as "reiniciar"

### CODE REVIEW

File: translations/base-es.yaml [hunk 1]
```diff
@@ -43,24 +43,24 @@ steamPage:
...
         [list]
-            [*] Modo historia en el que los edificios cuesten figuras
+            [*] Modo historia en el que los edificios tengan un coste de figuras
             [*] Más niveles y edificios (exclusivos del juego completo)
             [*] Mapas diferentes y tal vez obstáculos en el mapa
             [*] Configuración en la creación del mapa (Editar el número y tamaño de los recursos, la semilla, y más)
             [*] Más tipos de formas
             [*] Mejoras de rendimiento (Aunque el juego ya funciona muy bien!)
-            [*] Modo para daltónicos
+            [*] Modo para el daltonismo
```
Comment (K3rcus) [inline translations/base-es.yaml:63]: "Modo para daltónicos"  is the best traduction, "para el daltonismo" feels weird, is not a mode for the condition, but for the people who has the condition.

File: translations/base-es.yaml [hunk 2]
```diff
@@ -590,11 +589,11 @@ storyRewards:
     no_reward:
         title: Siguiente Nivel
         desc: >-
-            Este nivel no da recompensa, pero el siguiente si! <br><br> PS: Mejor no destruyas la fábrica que tienes - ¡Necesitarás <strong>todas</strong> esas figuras más adelante para <strong>desbloquear mejoras</strong>!
+            Este nivel no da recompensa, ¡pero el siguiente si! <br><br> PS: Mejor no destruyas la fábrica que tienes - ¡Necesitarás <strong>todas</strong> esas figuras más adelante para <strong>desbloquear mejoras</strong>!
     no_reward_freeplay:
         title: Siguiente Nivel
         desc: >-
-            ¡Felicidades! ¡Por cierto, más contenido está planeado para el juego completo!
+            ¡Felicidades! ¡Por cierto, hay más contenido está planeado para el juego completo!
```
Comment (K3rcus) [inline translations/base-es.yaml:596]: This is grammatically wrong, the right sentence is: "¡Por cierto, hay más contenido planeado para el juego completo!"
without the "está"

File: translations/base-es.yaml [hunk 3]
```diff
@@ -708,21 +707,19 @@ settings:
...
+            title: Modo para Daltonismo
+            description: Activa varias herramientas que permiten jugar si tienes alguna forma de daltonismo
         rotationByBuilding:
-          title: Rotation by building type
+          title: Rotación por tipo de edificio
           description: >-
-            Each building type remembers the rotation you last set it to individually.
-            This may be more comfortable if you frequently switch between placing
-            different building types.
+            Cada tipo de edificio recuerda la última rotación que seleccionaste individualmente. Esto puedo se mucho más cómodo si sueles cambiar a menudo entre colocar diferentes tipos de edificio.
```
Comment (K3rcus) [inline translations/base-es.yaml:722]: there are a mistake, is: "Esto puede ser mucho más cómodo si cambias a menudo entre colocar diferentes tipos de edificio."

"puede" with "E"
"ser" ended in r
"sueles" and "a menudo" is redundant

File: translations/base-es.yaml [hunk 4]
```diff
@@ -708,21 +707,19 @@ settings:
...
         disableCutDeleteWarnings:
             title: Deshabilitar las advertencias de Cortar/Eliminar
             description: >-
-                Deshabilita los dialogos de advertencia que se muestran cuando se cortan/eliminan mas de 100 elementos.
+                Deshabilita los diálogos de advertencia que se muestran cuando se cortan/eliminan mas de 100 elementos.
 
         enableColorBlindHelper:
-            title: Modo para Daltonicos
-            description: Activa varias herramientas que permiten jugar si eres daltonico.
+            title: Modo para Daltonismo
```
Comment (K3rcus) [inline translations/base-es.yaml:717]: Same as in line 63:

"Modo para daltónicos" is the best traduction, "para el daltonismo" feels weird, is not a mode for the condition, but for the people who has the condition.

look in any other software or just look in google for the 2 sentences: "para daltónicos" is the normal sentence in Spanish.

File: translations/base-es.yaml [hunk 5]
```diff
@@ -527,8 +527,7 @@ storyRewards:
 
     reward_painter:
         title: Pintor
-        desc: >-
-            The <strong>painter</strong> has been unlocked - Extract some color veins (just as you do with shapes) and combine it with a shape in the painter to color them!<br><br>PS: If you are colorblind, there is a <strong>color blind mode</strong> in the settings!
+        desc:  El <strong>pintor</strong> ha sido desbloqueado - ¡Extrae vetas de color (igual que lo haces con las figuras) y combínalas con una figura en el pintor para colorearlas! <br><br>PD: Si tienes alguna forma de daltonismo, ¡hay un <strong>modo daltonismo</strong> en las configuraciones!
```
Comment (K3rcus) [inline translations/base-es.yaml:530]: "modo daltonismo" should be change for "modo para daltónicos"

**Comment by tobspr** [comment]
I guess there are changes to do here?

**Comment by tobspr** [comment]
Closing in favour of #331

**Pull request closed without merging**